src/xma/src/xmaplugin/xmaplugin.cpp

```diff
@@ -560,7 +498,7 @@ XmaCUCmdObj xma_plg_schedule_work_item(XmaSession s_handle,
     cmd_obj_error.cmd_id2 = 0;
     cmd_obj_error.cmd_finished = false;
     cmd_obj_error.cu_index = -1;
-    cmd_obj_error.do_not_use1 = NULL;
+    cmd_obj_error.do_not_use1 = nullptr;
 
     XmaHwSessionPrivate *priv1 = (XmaHwSessionPrivate*) s_handle.hw_session.private_do_not_use;
     if (priv1 == NULL) {
```

Comment:
It would be best to check for a nullptr instead.

Reply:
done